source/recurring_payments/index.html.md.erb

```diff
@@ -19,6 +21,8 @@ Your service has [additional responsibilities when taking recurring payments](/r
 * telling users when you’re going to take or change payments 
 * letting users amend their recurring payments
 
+It would be helpful to familiarise yourself [the standard GOV.UK Pay payment flow](/payment_flow) before reading about recurring payments.
+
 This section outlines the recurring payment flow, your responsibilities, and how to set up and use agreements to take recurring payments.
```

Comment:
"This section explains"? I think "outlines" is a bit of a policy word, dangerously close to "sets out" :)

Reply:
'explains' is clearer English - good spot!